wiki/People/The_Team/ru.md

```diff
@@ -1,56 +1,95 @@
 
 ## Группы пользователей
 
-| имя | описание |
+Помимо перечисленных выше, за различными сторонами osu! следят специальные группы пользователей. Их можно опознать по особым цветам ников на форуме или в чате, а также титулам и значкам в профиле.
+
+| Имя | Описание |
 | :-- | :-- |
-| [Глобальные модераторы (GMT)](/wiki/People/Global_Moderation_Team) | Наблюдение за форумами и игровым чатом |
-| [Номинаторы карт (BNG)](/wiki/People/Beatmap_Nomination_Group) | Те, кто номинирует карты и днём и ночью |
-| [Разработчики](https://osu.ppy.sh/groups/11) | Реализация нововведений, исправление багов |
-| [osu!alumni](/wiki/People/osu!_Alumni) | Those known for their contributions who have since moved on |
-| [Техподдержка](/wiki/People/Support_Team) | Помощь в разделах Help и Feature Requests |
-| [osu!news](https://osu.ppy.sh/groups/25) | Качественные статьи с 2013 года |
-| [Турнирный персонал](https://osu.ppy.sh/groups/26) | Responsible for handling officially supported osu! tournaments across all gamemodes. |
-| [Контроль качества (QAT)](/wiki/People/Quality_Assurance_Team) | Контроль качества ранкнутых карт |
+| [Номинаторы](Beatmap_Nominators) | Номинирование на ранк достойных карт |
+| [Разработчики](Developers) | Добавление новых возможностей и исправление багов |
+| [Глобальные модераторы](Global_Moderation_Team) | Поддержание порядка на форуме и в чате |
+| [Команда оценки номинаций](Nomination_Assessment_Team) | Контроль деятельности номинаторов |
+| [osu! Alumni](osu!_Alumni) | Те, кто помогали osu!, но ушли на покой |
+| [Команда Project Loved](Project_Loved_Team) | Составление подборок карт, которые любит сообщество |
+| [Команда поддержки](Support_Team) | Решение пользовательских проблем и рассмотрение новых идей |
+
+## Неактивная часть команды
+
+| Имя | Роль |
+| :-- | :-- |
+| ![][flag_CA] [awp](https://osu.ppy.sh/users/2650) | Смотритель за порядком, чиби-артист. Автор веб-комиксов [Barrel Roll Weekly](http://brw.twinkfish.com/) |
+| ![][flag_NO] [cYsmix](https://osu.ppy.sh/users/272870) | Аудио-дизайнер, [официальный Featured Artist](https://osu.ppy.sh/beatmaps/artists/2) |
+| ![][flag_NZ] [deadbeat](https://osu.ppy.sh/users/128370) | Организатор и администратор международных турниров |
+| ![][flag_US] [Derekku](https://osu.ppy.sh/users/91341) | Смотритель за порядком, менеджер сообщества |
+| ![][flag_NZ] [Echo](https://osu.ppy.sh/users/431) | Разработчик osu!, автор IRC-сервера osu!Bancho, администратор сайта. [Личный блог](http://blog.echo.sh/) |
+| ![][flag_US] [HappyStick](https://osu.ppy.sh/users/256802) | Ведущий osu! Coffee Hour, стример международных турниров, организатор турниров |
+| ![][flag_NL] [Intermezzo](https://osu.ppy.sh/users/136842) | Разработчик osu!, автор реализации osz2 и p2p-бэкенда osu! |
+| ![][flag_US] Jim | Дизайнер самой первой версии сайта, хостер osu! в ранние дни. Персональный проект: [Brand New Games](http://www.bravegamer.com/) |
```

Comment:
line was updated recently, at least the link text should be different

Reply:
ugh, I missed a part of the fix I made myself. thanks for catching